src/pkg/runner/runner.go

```diff
@@ -168,14 +186,49 @@ func (r *Runner) getTask(taskName string) (types.Task, error) {
 	return types.Task{}, fmt.Errorf("task name %s not found", taskName)
 }
 
+func mergeEnv(env1, env2 []string) []string {
```

Comment:
Can you toss a comment on this fn indicating that it merges `env1` into`env2`?

Reply:
was just about to do that as this function is very ambiguous